### DISCUSSION TITLE
user-guide/mobile

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#181] [文档评论区]
# user-guide/mobile

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/mobile.html

**Comment by YehKai222** [comment]
zoo for zotero 卡的要死，已卸

**Comment by dingtiany** [comment]
有大佬指教一下吗，我在mac设置了WebDAV同步，并且验证通过了，但是在我pad上准备下载pdf的时候却是很久很久都不能pdf下载下来。用的是lnfiniCloud的WebDAV

**Comment by aucu** [comment]
请问官方安卓版本（1）支持在手机上阅读电脑端pdf上的注释吗？（2）支持在手机上给pdf附件做注释，并同步保留，可在电脑端查看吗？（3）官方版本啥时候能有个华为Huawei鸿蒙版本？

谢谢各位大佬解答

**Reply by HungrySemiconductor** [reply]
`安卓版本：1.0.0-119`
`WebDAV：坚果云`
<br>安卓版本移动端和电脑端相互之间可以实现同步，但实时同步有些不稳定。
<br>PC端修改后，安卓端没有手动同步的按钮，自动同步会稍有延时；安卓端修改后，PC端点击右上角同步按钮，可以实时同步。

**Reply by northword** [reply]
> 安卓端没有手动同步的按钮，自动同步会稍有延时

@HungrySemiconductor 安卓端在条目列表下拉可以出发同步。

**Reply by Moemoemaru** [reply]
鸿蒙直接用安卓版就行，虽然没法用googleplay装，但我用up to down装好了，使用官方同步就可以

**Comment by SolaProject** [comment]
`安卓版本：1.0.0-186`
`WebDAV：阿里云自建服务器 Docker 镜像 bytemark/webdav`

通过电脑端可以实现 WebDAV 同步，但是在安卓端验证服务时提示 `Could not connect to WebDAV server`，在安卓端浏览器连接远程服务器的 WebDAV 可以正常访问。（同时测试了 InfiniCloud 的 WebDAV 可以正常同步）

**Comment by Swordsman1999** [comment]
小文件可以同步，大文件同步不了？

**Comment by 74Smith** [comment]
各位大佬，怎么转换为中文界面啊？

**Comment by Swordsman1999** [comment]
可能没收到，请微信一下！

**Comment by Swordsman1999** [comment]
可能没收到，请微信一下！

**Comment by 74Smith** [comment]
我已经通过谷歌商店搞好了，谢谢



---Original---
From: ***@***.***&gt;
Date: Fri, Apr 3, 2026 13:14 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [zotero-chinese/wiki] user-guide/mobile (Discussion #181)




 
鸿蒙直接用安卓版就行，虽然没法用googleplay装，但我用up to down装好了，使用官方同步就可以

**Comment by 74Smith** [comment]
不用了，谢谢



---Original---
From: ***@***.***&gt;
Date: Fri, Apr 3, 2026 13:14 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [zotero-chinese/wiki] user-guide/mobile (Discussion #181)




 可能没收到，请微信一下！
